src/components/inboxComponent/inboxMessageComponent/_test_/inboxMessageComponent.spec.tsx

```diff
@@ -0,0 +1,33 @@
+import { fireEvent, render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import '@testing-library/jest-dom';
+//import userEvent from '@testing-library/user-event';
+
+import InboxMessageComponent from '../inboxMessageComponent';
+
+describe('InboxMessageComponent Component', () => {
+  beforeEach(() => {
+    render(
+      <MemoryRouter>
+        <InboxMessageComponent id="" title="" description="" />
```

Comment:
Do you need these props? Or you can delete them?

Reply:
These props are nessesary according to the syntax 
<img width="555" alt="image" src="https://github.com/VitalikMatvieiev/edu-platform/assets/151072156/6779d4be-e598-4075-9869-15de00bc7646">